partcad/src/partcad/part_factory_obj.py

```
@@ -0,0 +1,152 @@
+class PartFactoryObj(PartFactoryFile):
+    # Constants
+    MIN_SIMPLE_INFLIGHT = 1
+    MIN_SUBPROCESS_FILE_SIZE = 64 * 1024  # 64 KB
+    PYTHON_RUNTIME_VERSION = "3.10"
+
+    # Shared counters for inflight operations
+    lock = threading.Lock()
+    count_inflight_simple = 0
+    count_inflight_subprocess = 0
+
+    def __init__(self, ctx, source_project, target_project, config):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)